### ISSUE TITLE
Implement support enumaration

### ISSUE THREAD

**Issue by theref** [theref/sage-game-theory#46]

**Comment by drvinceknight** [comment]
Cool: this is waiting on Ted to implement as well yes?

**Comment by theref** [comment]
He has `ExternalEnumPureSolver`, `ExternalEnumMixedSolver` and `ExternalEnumPolySolver`. I don't know which one it is that we need but yes, at some point these should be implemented within Gambit.

**Comment by drvinceknight** [comment]
Cool.

**Comment by theref** [comment]
Note: Our own implementation should contain a warning when games get above a certain size. This will probably need to be tested so we can agree on an appropriate size.

**Comment by theref** [comment]
i think this is basically done now, i'll create a new ticket for my above comment

**Comment by drvinceknight** [comment]
Let's just keep it open until I've taken a good look at it (I think it's pretty much done but I don't want to forget that's all). Open the other ticket as well.

**Comment by drvinceknight** [comment]
This commit bf096392bd957686784aea75bbb781eece40a174 shows some code that's commented out in which I took a look at solving a single linear system instead of two. Tests took twice as long to run. This could be because of the actual way the matrices are being constructed and once I've re factored that it could be worth going back to but I don't think so.
